egs/hkust/s5/local/nnet3/tuning/run_tdnn_1a.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/bash
+
+# exp 1a: original baseline
+#         (deprecated because since Kaldi 5.2 make_configs.py is no longer usable)
+
+# At this script level we don't support not running on GPU, as it would be painfully slow.
+# If you want to run without GPU you'd have to call train_tdnn.sh with --gpu false,
+# --num-threads 16 and --minibatch-size 128.
+set -e
+
+stage=0
+nj=10
+train_stage=-10
+affix=
+common_egs_dir=
+
+# training options
+initial_effective_lrate=0.0015
+final_effective_lrate=0.00015
+num_epochs=4
+num_jobs_initial=2
+num_jobs_final=12
+remove_egs=true
+
+# feature options
+use_ivectors=true
+
+# End configuration section.
+
+. ./cmd.sh
+. ./path.sh
+. ./utils/parse_options.sh
+
+if ! cuda-compiled; then
+  cat <<EOF && exit 1
+This script is intended to be used with GPUs but you have not compiled Kaldi with CUDA
+If you want to use GPUs (and have them), go to src/, and configure and make on a machine
+where "nvcc" is installed.
+EOF
+fi
+
+dir=exp/nnet3/tdnn_sp${affix:+_$affix}
+gmm_dir=exp/tri5a
+train_set=train_sp
+ali_dir=${gmm_dir}_sp_ali
+graph_dir=$gmm_dir/graph
+
+if [ $stage -le 0 ]; then
+  local/nnet3/run_ivector_common.sh --stage $stage \
+    --ivector-extractor exp/nnet3/extractor || exit 1;
+fi
+
+if [ $stage -le 8 ]; then
+  echo "$0: creating neural net configs";
+
+  # create the config files for nnet initialization
+  python steps/nnet3/tdnn/make_configs.py  \
```

Comment:
is this the script you meant to put as 1a?  this looks like the old, non-working one.
I was suggesting to just discard that and put the current best one.  but I guess you have not finished tuning.

Reply:
Oh sorry...It's in the plan to remove it. Haven't push the changes to the online repo though